radixdlt/src/main/java/com/radixdlt/consensus/VertexInsertionException.java

```diff
@@ -0,0 +1,14 @@
+package com.radixdlt.consensus;
+
+/**
+ * An exception indicating a failure in inserting a vertex into a VertexStore
+ */
+class VertexInsertionException extends Exception {
+	VertexInsertionException(String message) {
+		super(message);
+	}
+
+	VertexInsertionException(Exception cause) {
```

Comment:
This constructor should require a context message as well.  If just the cause is included, it can be hard to determine what went wrong without a context message.